adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureParser.scala

```diff
@@ -118,7 +122,11 @@ class GTFParser extends FeatureParser {
     _id.foreach(f.setFeatureId)
     _parentId.foreach(parentId => f.setParentIds(List[String](parentId)))
 
-    f.setAttributes(attrs)
+    if (frame != ".") {
+      f.setAttributes(collection.mutable.Map(attrs.toSeq: _*) += ("frame" -> frame))
```

Comment:
Also, could we get some docs here? What's this conditional for? It looks like we're special casing some formatting thing. Why's that? Etc.